analysis/wwz/make_datacards.py

```diff
@@ -220,6 +236,22 @@ def get_rate_systs(proc_lst):
 
     return out_dict
 
+def determine_updo_same(nom,up,down):
+    if nom < 0:
+        raise Exception("Negative values should have been fixed by this point!")
+    elif ((up > nom) and (down > nom)):
+        return True
+    elif ((up < nom) and (down < nom)):
+        return True
+    else: return False
+
+def fix_updown_same(nom,up,down):
```

Comment:
Add a comment before this function that explains what it is and what it does

Reply:
Done